lib/commands/project.js

```diff
@@ -65,6 +65,81 @@ function sortImages(images) {
   return images;
 }
 
+project.resize = function (projectName, servoSize, cb) {
+  var projectId, projectName;
+
+  projectController.find({
+      userId : userConfig.data.userId
+    },
+    function (err, projects) {
+      if (err) {
```

Comment:
Even though this code is replicated at other places, it def has some problems. I'll take a deeper look into seeing how to fix this up. But this is fine for now.